monai/engines/evaluator.py

```diff
@@ -67,8 +67,8 @@ class Evaluator(Workflow):
             for more details, check: https://pytorch.org/ignite/generated/ignite.engine.engine.Engine.html
             #ignite.engine.engine.Engine.register_events.
         decollate: whether to decollate the batch-first data to a list of data after model computation,
-            default to `True`. if `False`, postprocessing will be ignored as the `monai.transforms` module
-            assumes channel-first data.
+            default to `True`. if `False`, as the `monai.transforms` module assumes channel-first data,
+            please don't set `monai.transforms` for `postprocessing`.
```

Comment:
how about:
```suggestion
            default to `True`. We recommend `decollate=True` when `postprocessing` uses components from 
           `monai.transforms`.
```

Reply:
Sure, updated in the latest commit.
Thanks.